frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -101,27 +101,27 @@ export const AboutResourcePage = ({
 
   // States to store the different input values
   const [title, setTitle] = useState<SupportedLanguageKey<string>>(
-    resourceData.title ?? emptyLangauges
+    resourceData.title ?? emptyLangauges,
```

Comment:
It's not related to your changes, but I see that there is a small typo error here: `emptyLangauges`, should be `emptyLanguages`